pom.xml

```diff
@@ -72,6 +84,10 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
```

Comment:
it's added by default, no need to declare here if it's not re-configured

Reply:
replaced